packages/tests/server/adapters/awsLambda.test.tsx

```diff
@@ -92,7 +92,7 @@ test('basic test', async () => {
   expect(result).toMatchInlineSnapshot(`
     Object {
       "headers": Object {
-        "Content-Type": "application/json",
+        "content-type": "application/json",
```

Comment:
lambda always reformats as 'Content-Type' in the wild, though not sure if that's a problem here?